src/v/config/configuration.cc

```diff
@@ -1403,7 +1403,78 @@ configuration::configuration()
       "Time interval between two node status messages. Node status messages "
       "establish liveness status outside of the Raft protocol.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
-      100ms) {}
+      100ms)
+  , enable_controller_log_rate_limiting(
+      *this,
+      "enable_controller_log_rate_limiting",
+      "Enables limiting of controller log write rate",
+      {.needs_restart = needs_restart::no, .visibility = visibility::user},
```

Comment:
I think that sometimes users want to turn it off.
If they want to create some burst traffic (e.c. initialization of cluster) it would be more comfortable to disable feature and then reenable it than chaning limits and than chaning it back.
Maybe we should just turn this setting on by default.
@mmaslankaprv What do you think about it?

Reply:
I see. We'd have to document this in that case.